core/tests/test_utils.py

```diff
@@ -164,10 +196,21 @@ def signup_superadmin_user(self):
     def log_line(self, line):
         """Print the line with a prefix that can be identified by the
         script that calls the test.
+
+        Args:
+            line: string. Contains log information which will be
```

Comment:
"string" isn't a valid Python type.

Ditto below (multiple places).

Reply:
changed it to str

Reply:
changed it to str